DogeNet.BLL/Features/Account/CreateAccount/CreateAccountHandler.cs

```diff
@@ -28,18 +28,20 @@ public CreateAccountHandler(DBContext context, IUserManagerService userManager,
             this.mapper = mapper;
         }
 
-        public async Task<int> Handle(CreateAccountCommand request, CancellationToken cancellationToken)
+        public async Task<Unit> Handle(CreateAccountCommand request, CancellationToken cancellationToken)
         {
             var user = this.mapper.Map<IdentityUser>(request.model);
 
             var result = await this.userManager.CreateIdentityUser(user, request.model.Password);
 
             var appUser = this.mapper.Map<User>(request.model);
+            appUser.IdentityId = user.Id;
+            appUser.CreatedAt = System.DateTime.Now;
```

Comment:
Need to move this into mapper

Reply:
IdentityId i cannot map in profile cause it is generating in handler when I create IsentityUser instance, but I can map datetime in profile